packages/primitives/image/src/Root.native.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { component, startWithType, mapDefaultProps } from 'refun'
+import { Image as ImageRN } from 'react-native'
+import { TImage } from './types'
+
+const getImageSourceFromType = (source: any) => (typeof source === 'string' ? { uri: source } : source)
```

Comment:
according to types it's always a string, and I believe it should be so :) so on native we just always normalize it to an `uri` object

Reply:
mmmmmm maybe i should change the type than....
or we do not want to let people do:
```
source={{
    uri: 'https://potato.com/pepes/feels.png',
    method: 'POST',
    headers: {
      Pragma: 'no-cache',
    },
    body: 'Your Body goes here',
  }}
```

Reply:
@deepsweet WDYT?